VideoWeb/VideoWeb/ClientApp/src/app/shared/toast/room-closing/room-closing-toast.component.html

```diff
@@ -0,0 +1,6 @@
+<ng-container>
+<div class="call" style="display:block;" id="room-closing">
+    <p class="govuk-body" style="font-size:0.95em;"><strong>Alert</strong><br><span id="room-closing-alert-message">This room will close in {{durationStr}}.</span></p>
+    <div class="small-button" style="float:right; cursor:pointer;" (click)="dismiss()">Dismiss</div>
```

Comment:
remove inline styling & use flex instead of floats

Reply:
Done